feature-libs/cart/quick-order/styles/_quick-order-form.scss

```diff
@@ -5,8 +5,9 @@ cx-quick-order-form {
   }
 
   .quick-order-form-input {
-    align-items: center;
+    align-items: flex-start;
     display: flex;
+    flex-direction: column;
```

Comment:
When the viewport is mobile, the alignment of the input and button changes is wrong

-----
What are the point of these changes? They don't do anything, as well as being `flex`.

Please make sure Mikel or Miguel looks at the entire styling right now so it doesn't get pushed last minute. I'm sure they haven't taken a look yet as the classes were prepended with `cx-`

Reply:
Styles changed.

Reply:
I'm sure we have to add cx- on most of our classes espcially features that creates new ones